### PR TITLE
test: switch from travis-ci to GitHub CI

### PR DESCRIPTION
**Pull request by divdavem**
Before submitting a pull request, please make sure you have at least performed the following:

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md) and [DEVELOPER.md](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/DEVELOPER.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated any applicable tests.
 - [x] added/updated any applicable API documentation.
 - [x] added/updated any applicable demos.

### CODE REVIEW

**Comment by divdavem** [comment]
Now that #4005 is integrated and this PR rebased, tests are green: https://github.com/divdavem/ng-bootstrap/actions/runs/561460926
This PR is ready to be reviewed

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      SAUCE_ACCESS_KEY: c5ea8b730bc9-378b-43e4-6e34-e66aa371
+      BROWSER_PROVIDER_READY_FILE: /tmp/sauce-connect-ready
+      LOGS_DIR: /tmp/logs
+    steps:
+      - uses: actions/checkout@v2
+      - run: mkdir -p $LOGS_DIR && ./scripts/sauce_connect_setup.sh
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - uses: microsoft/playwright-github-action@v1
```
Comment (maxokorokov) [inline .github/workflows/ci.yml:18]: oh, this simplifies things significantly :D

**Review by maxokorokov — APPROVED** [review]
Hey, @divdavem, LGTM! Thanks!

**Comment by maxokorokov** [comment]
@divdavem, should we merge it at this point?

**Comment by divdavem** [comment]
> @divdavem, should we merge it at this point?

Yes I think it is a good idea to merge it now.

**Pull request merged**